src/V3EmitC.cpp

```diff
@@ -207,6 +207,113 @@ class EmitCImp final : EmitCFunc {
             emitCFuncDecl(funcp, modp);
         }
     }
+    void emitVarDecls(const AstNodeModule* modp) {
+        // Output a list of variable declarations
+
+        std::vector<std::pair<const AstVar*, VarClass>> varList;
+        bool lastAnon = false;  // initial value is not important, but is used
+
+        const auto emitCurrentList = [this, &varList, &lastAnon]() {
+            if (varList.empty()) return;
+
+            const auto emitDecl = [this](const std::pair<const AstVar*, VarClass>& pair) {
+                static const char* prefix[]
+                    = {"/* PORT      */ ", "/* SIGNAL    */ ", "/* INTERNAL  */ ",
+                       "/* PARAMETER */ ", "@-ERROR-OTHER   "};
```

Comment:
I don't like the looks of this being first in the output.  Perhaps just leave it out?  Or add as a comment at the end of the declaration line?

Reply:
Ok, dropped these as they are not particularly informative.